python/tests/e2e/cassettes/structured_output/anthropic/async.yaml

```diff
@@ -5,9 +5,10 @@ interactions:
       respond to the user''s query using the __mirascope_formatted_output_tool__ tool
       for structured output.","tool_choice":{"type":"tool","name":"__mirascope_formatted_output_tool__","disable_parallel_tool_use":true},"tools":[{"name":"__mirascope_formatted_output_tool__","description":"Use
       this tool to extract data in Book format for a final response.\nA book with
-      a rating. The title should be in all caps!","input_schema":{"description":"A
-      book with a rating. The title should be in all caps!","properties":{"title":{"title":"Title","type":"string"},"author":{"title":"Author","type":"string"},"rating":{"description":"For
```

Comment:
nb (here and elsewhere): This fixes an issue I've noted in the past, where we would include the description twice. ("The title should be in all caps!" is present twice in the left and only once on the right.)